.github/workflows/ci.yml

```diff
@@ -125,11 +129,11 @@ jobs:
       - name: Install FuseML
         run: |
           ./dist/fuseml-installer version
-          make fuseml-install-with-extensions
+          make fuseml-install-with-kfserving
```

Comment:
I think here you want `make fuseml-install-with-${{ matrix.target }}` ?

Reply:
ah, missed that one